openslides_backend/models/meeting.py

```diff
@@ -16,10 +18,254 @@ class Meeting(Model):
+            "users_pdf_url",
+            "users_pdf_wlan_ssid",
+            "users_pdf_wlan_password",
+            "users_pdf_wlan_encryption",
+            "users_email_sender",
+            "users_email_replyto",
+            "users_email_subject",
+            "users_email_body",
+        ]
+
     id = fields.IdField(description="The id of this meeting.")
     committee_id = fields.RequiredForeignKeyField(
         description="The id of the committee of this meeting.",
         to=Collection("committee"),
         related_name="meeting_ids",
     )
-    name = fields.RequiredCharField(description="The name of this meeting.")
+
+    welcome_title = fields.CharField()
+    welcome_text = fields.TextField()
+
+    name = fields.CharField(maxLength=100)
+    description = fields.CharField(maxLength=100)
+
+    date = fields.TextField()
+    location = fields.TextField()
+    start_time = fields.TimestampField()
+    end_time = fields.TimestampField()
+    # custom_translations = JSON  # TODO
+
+    url_name = fields.TextField()
+    is_template = fields.BooleanField()
+    enable_anonymous = fields.BooleanField()
+
+    conference_show = fields.BooleanField()
+    conference_auto_connect = fields.BooleanField()
+    conference_los_restriction = fields.BooleanField()
+    conference_stream_url = fields.TextField()
+
+    projector_default_countdown_time = fields.IntegerField()
+    projector_countdown_warning_time = fields.IntegerField(minimum=0)
+
+    export_csv_separator = fields.TextField()
+    export_csv_encoding = fields.TextField(enum=["utf-8", "iso-8859-15"])
+    export_pdf_pagenumber_alignment = fields.TextField(enum=["left", "right", "center"])
+    export_pdf_fontsize = fields.IntegerField(enum=[10, 11, 12])
+    export_pdf_pagesize = fields.TextField(enum=["A4", "A5"])
+
+    agenda_show_subtitles = fields.BooleanField()
+    agenda_enable_numbering = fields.BooleanField()
+    agenda_number_prefix = fields.TextField(maxLength=20)
+    agenda_numeral_system = fields.TextField(enum=["arabic", "roman"])
+    agenda_item_creation = fields.TextField(
+        enum=["always", "never", "default_yes", "default_no"]
+    )
+    agenda_new_items_default_visibility = fields.IntegerField(enum=[1, 2, 3])
+    agenda_show_internal_items_on_projector = fields.BooleanField()
+
+    list_of_speakers_amount_last_on_projector = fields.IntegerField(minimum=0)
+    list_of_speakers_amount_next_on_projector = fields.IntegerField(minimum=-1)
+    list_of_speakers_couple_countdown = fields.BooleanField()
+    list_of_speakers_show_amount_of_speakers_on_slide = fields.BooleanField()
+    list_of_speakers_present_users_only = fields.BooleanField()
+    list_of_speakers_show_first_contribution = fields.BooleanField()
+
+    motions_default_workflow_id = fields.RequiredForeignKeyField(
+        to=Collection("workflow"), related_name="default_workflow_meeting_id",
+    )
+    motions_default_statute_amendments_workflow_id = fields.RequiredForeignKeyField(
```

Comment:
Please remove this setting here and in the list above. We should define this as boolean field in workflow model.